katsdpimageutils/test/test_primary_beam_correction.py

```diff
@@ -0,0 +1,111 @@
+"""Tests for :mod:`katsdpimageutils.primary_beam_correction`."""
+import shutil
+import os
+
+import numpy as np
+from astropy.io import fits
+from nose.tools import assert_equal
+
+from ..primary_beam_correction import _get_value_from_history, weighted_average, inverse_variance, standard_deviation, write_new_fits # noqa E501 
+
+
+class TestStandardDeviation():
```

Comment:
Don't need the brackets:

```suggestion
class TestStandardDeviation:
```

Reply:
fixed